.eslintrc.json

```diff
@@ -12,7 +12,14 @@
       "components": [ "Link" ],
       "specialLink": [ "hrefLeft", "hrefRight", "to" ],
       "aspects": [ "noHref", "invalidHref", "preferButton" ]
-    }]
+    }],
+    "jsx-a11y/label-has-for": [ 2, {
+      "components": [ "Label" ],
+      "required": {
+          "some": [ "nesting", "id" ]
+      },
+      "allowChildren": false
+  }]
   }
```

Comment:
You have added this to bypass the rule that every label should have an htmlFor with the id of its corresponding input field. Are you sure this rule should be bypassed? [This](https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/51) seems to be a discussion about this rule.

Reply:
yeah  i can link the label to the textarea form control in my case but that is not really required. That is why I added the rule.